20240518C9/Practice.py

```diff
@@ -0,0 +1,9 @@
+foodlist = ['tacos']
+1=='order'
+2=='exit'
+while True:
+    i=input()
+    if i=='1':
+        print("you ordered" + str[foodlist])
```

Comment:
Use index to access 'tacos'! Also 'tacos' is already a string type so str() is unnecessary